proxy/ptunnel/commands/ssh.go

```diff
@@ -159,6 +159,6 @@ func NewSSH(base *client.BaseFlags) *SSH {
 	root.Command.Flags().StringVar(&root.Subcommand, "tunnel-command", tcommand, "Subcommand to use with the tunnel command. Defaults to empty if the tunnel command does not end with enkit")
 	root.Command.Flags().StringVar(&root.Extra, "tunnel-extra", "", "Extra arguments to pass to the tunnel command")
 	root.Command.Flags().BoolVar(&root.UseInternalAgent, "use-internal-agent", true, "Use the builtin agent that enkit manages")
-
+	root.Command.AddCommand(NewAgentCommand(root.BaseFlags))
```

Comment:
meta: leave as is for now, but I'm not convinced that adding this to `ptunnel` as `agent` subcommand is the right thing to do. 2 concerns:
1) Agent management has to do with authentication, it's weird to see this command in a library that does tunneling.
2) Let's say in my network I have two hosts: one named "agent", the other named "plumber". I can `enkit ssh plumber` just fine (all works) while I'll get a fairly unexpected result if I run `enkit ssh agent`. It'd take me a few minutes to realize that `enkit ssh -- agent` is the correct command, != from `enkit ssh agent`, and different from `enkit ssh agent -- rsync -avz`.

Solutions that come to mind:
a) make agent a  top level command. We have `enkit ssh`, `enkit login`, we can have `enkit agent`. This solves both 1 and 2.
b) make agent entirely flags based. Eg, like you do with `-l` and `-e`, we can just have `-r` to run a command (as proposed above, runs all args with os/exec). So the ssh command itself is modified to have 3 extra flags.

Reply:
So the reason for putting this here is that in my head I know its an ssh agent,  dumb logic of enkit => ssh =>  agent. I suppose that's not as transparent to some people. and you're right that the way the cli is currently designed does use kwargs, which means it should be a terminal command. 

Moving it to top level

Reply:
FWIW, yes, putting it under ssh was the logical choice. Had I thought about this a while back, probably I would have made `ssh` have subcommands, like `enkit ssh run`.